src/crypto/CHIPCryptoPALmbedTLS.cpp

```diff
@@ -496,6 +520,22 @@ mbedtls_ecp_group_id MapECPGroupId(SupportedECPKeyTypes keyType)
     }
 }
 
+#if defined(MBEDTLS_USE_TINYCRYPT)
+
+static inline mbedtls_uecc_keypair * to_keypair(P256KeypairContext * context)
+{
+    return SafePointerCast<mbedtls_uecc_keypair *>(context);
+}
+
+static inline const mbedtls_uecc_keypair * to_const_keypair(const P256KeypairContext * context)
+{
+    return SafePointerCast<const mbedtls_uecc_keypair *>(context);
+}
+
+#else
+
+#if defined(MBEDTLS_ECP_C)
```

Comment:
Is it needed? If `MBEDTLS_USE_TINYCRYPT` is not defined then ECP_C must be enabled in mbedtls, right?